src/app/portlets/content-types/fields/service/field-drag-drop.service.ts

```diff
@@ -72,6 +72,9 @@ export class FieldDragDropService {
         this._fieldDropFromTarget = dragulaService.dropModel().pipe(
             tap(() => {
                 this.draggedEvent = true;
+                setTimeout(() => {
+                    this.draggedEvent = false;
```

Comment:
Because this is mock in the tests, if I came here and remove this line, will the test break?

Reply:
yes